front50/front50-s3/src/main/java/com/netflix/spinnaker/front50/config/S3ClientFactory.java

```diff
@@ -72,14 +72,10 @@ public static S3Client create(
 
     if (!StringUtils.isEmpty(s3Properties.getEndpoint())) {
       s3Builder.endpointOverride(URI.create(s3Properties.getEndpoint()));
-
-      if (!StringUtils.isEmpty(s3Properties.getRegionOverride())) {
```

Comment:
Is it time to remove regionOverride and signerOverride from [S3BucketProperties](https://github.com/spinnaker/spinnaker/blob/d0f0620ac1d415e4c23f6fd8e31a4f661331a2a4/front50/front50-s3/src/main/java/com/netflix/spinnaker/front50/config/S3BucketProperties.java#L24-L27) ?  It doesn't look like they're used.

Reply:
There is [S3PluginStorageProperties](https://github.com/spinnaker/spinnaker/blob/d0f0620ac1d415e4c23f6fd8e31a4f661331a2a4/front50/front50-s3/src/main/java/com/netflix/spinnaker/front50/config/S3PluginStorageProperties.java) that extends S3BucketProperties which may get impacted. But as the AmazonS3 itself is upgrades to S3Client, any plugin would have been broken already. So shall I go ahead and remove those or wait for some more time?

Reply:
I'd say remove them, and say we did in a release note.